app/views/claims/verified.html.erb

```diff
@@ -21,9 +21,14 @@
         </span>
       </summary>
       <div class="govuk-details__text">
-        If you find that some of this information is incorrect please contact support at
-        <%= mail_to support_email_address(current_policy_routing_name), support_email_address(current_policy_routing_name), class: "govuk-link" %>
-        as this could affect your application.
+        <p class="govuk-body">
+          If this information is out of date you do not need to tell us. This will not affect your claim.
+        </p>
+        <p class="govuk-body">
+          If you do not recognise some of these details please let us know by emailing
+          <%= mail_to support_email_address(current_policy_routing_name), support_email_address(current_policy_routing_name), class: "govuk-link" %>.
```

Comment:
I don't think you need the second argument if you want to show the email address as the link text do you?

Reply:
Yeah, you're right. If the second argument is `nil` it will use the link as the text. Updated